lib/internal/modules/cjs/loader.js

```diff
@@ -966,10 +967,12 @@ function getExportsForCircularRequire(module) {
  * 3. Otherwise, create a new module for the file and save it to the cache.
  *    Then have it load the file contents before returning its exports object.
  * @param {string} request Specifier of module to load via `require`
- * @param {string} parent Absolute path of the module importing the child
+ * @param {Module} parent Absolute path of the module importing the child
  * @param {boolean} isMain Whether the module is the main entry point
  */
 Module._load = function(request, parent, isMain) {
+  const start = performance.now();
```

Comment:
`performance.now()` in itself leads to an overhead. I think we need to guard these behind a condition or a flag, or invent a specifier in `debuglog()` that leads to timing computation only when enabled.

Also as the test failure indicates this now load a bunch more perf-hooks modules unconditionally which slows down startup.